parts/k8s/addons/1.16/ip-masq-agent.yaml

```diff
@@ -38,8 +38,6 @@ spec:
       - name: azure-ip-masq-agent
         image: {{ContainerImage "ip-masq-agent"}}
         imagePullPolicy: IfNotPresent
-        args:
-          - --enable-ipv6={{ContainerConfig "enable-ipv6"}}
```

Comment:
Why are we removing the `--enable-ipv6` flag? This is required for dual-stack and IPv6 single stack clusters. This flag is supported from masq agent release `2.5.0` which is the current default in aks-engine.